src/screens/CreatePost/elements/SheetAddTopic.js

```diff
@@ -6,13 +6,14 @@ import {ScrollView, StyleSheet, Text, TextInput, TouchableNativeFeedback, View}
 import Card from './Card';
 import TopicItem from '../../../components/TopicItem';
 import {Button} from '../../../components/Button';
-import {colors} from '../../../utils/colors';
 import {convertString} from '../../../utils/string/StringUtils';
 import {fonts} from '../../../utils/fonts';
 import {getTopics} from '../../../service/topics';
 import {isEmptyOrSpaces} from '../../../utils/Utils';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been updated to use the `COLORS` constant from the `utils/theme` module instead of the previous `colors` constant. This change is consistent with the PR summary and seems to be a part of a larger refactoring effort to standardize color usage across the codebase.
```diff
- import {colors} from '../../../utils/colors';
+ import {COLORS} from '../../../utils/theme';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->